index.html

```diff
@@ -53,34 +64,35 @@ <h1 class="display-4">Documentation</h1>
 
 </head>
 <body>
-    
-  <div class="container justify-content-md-center">
-    <script async src="https://cse.google.com/cse.js?cx=010028121289637253157:oyhvms8bdqm"></script>
+    <div class="container justify-content-md-center">
+    <script async src="https://cse.google.com/cse.js?cx=001595182637908644242:njcczvsmzd4"></script>
```

Comment:
Where does these keys come from?

Reply:
The keys come from the Google Programmable Search Engine [https://programmablesearchengine.google.com/](https://programmablesearchengine.google.com/).

Reply:
Use this one:
<script async src="https://cse.google.com/cse.js?cx=005501767810126761962:bnqqnli8vek"></script>
<div class="gcse-search"></div>